api/proto/auth/client.go

```diff
@@ -32,7 +30,7 @@ var log = logrus.WithFields(logrus.Fields{
 type Client struct {
 	sync.Mutex
 	ClientConfig
-	grpc auth.AuthServiceClient
+	grpc AuthServiceClient
```

Comment:
Maybe we should just embed the client here. Not sure if there is going to be a huge benefit other than writing boilerplate code if we don't.